composeCore/src/main/java/com/hamdan/forzenbook/compose/core/theme/Theme.kt

```diff
@@ -72,19 +72,25 @@ object ForzenbookTheme {
             Dimens(
                 SmallGrid,
                 borderDimensionsStatic,
-                NormalGrid,
                 SmallImages
             )
         } else {
             // average
             Dimens(
                 NormalGrid,
                 borderDimensionsStatic,
-                NormalGrid,
                 NormalImages
             )
         }
 
+    val ForzenbookTheme.staticDimens: Dimens
```

Comment:
Make the extensions off MaterialTheme

Reply:
Do you want me to make the other dimens and typography do the same or just specificaly the static dimens?